prog/vm/prep_host.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 class Prog::Vm::PrepHost < Prog::Base
-  subject_is :sshable
+  subject_is :sshable, :vm_host
 
   label def start
-    sshable.cmd("sudo host/bin/prep_host.rb")
+    sshable.cmd("sudo host/bin/prep_host.rb " + vm_host.ubid)
```

Comment:
nit: 
1. We usually `shellescape` args passed in a shell command, e.g. ` + vm_host.ubid.shellescape`. This avoids bugs if an arg is empty or has whitespace or anything that might have a different interpretation in the shell parser.
2. We usually use string interpolation to embed variables, rather than using `+`. That is: `"sudo host/bin/prep_host.rb #{vm_host.ubid.shellescape}"`

Both of the above points are minor here, as what you did will most likely work fine, these are just the rules we usually follow.